### PR TITLE
raindrop-io: add ability to search title only

### PR DESCRIPTION
**Pull request by lardissone**
## Description

Added ability to search only on title by adding a preference option.

Suggested on these issues: [#13867](https://github.com/raycast/extensions/issues/13867) and [#8939](https://github.com/raycast/extensions/issues/13867)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/raindrop-io/package.json [hunk 1]
```diff
@@ -8,11 +8,7 @@
   "author": "lardissone",
   "contributors": [
     "shubham",
-    "pernielsentikaer",
-    "sh-cho",
-    "tdkn",
-    "j3lte",
-    "rien7"
+    "pernielsentikaer"
```
Comment (pernielsentikaer) [inline extensions/raindrop-io/package.json:11]: Why is there SO many changes and removed contributors here?

Comment (lardissone) [reply extensions/raindrop-io/package.json:11]: hmm, weird, I didn't changed this. I think the pull-contributors command broke it.

Let me restore these changes.

Comment (pernielsentikaer) [reply extensions/raindrop-io/package.json:11]: Thanks 🙂

Comment (lardissone) [reply extensions/raindrop-io/package.json:11]: Will close this PR and create a new one. Something was wrong with my outdated fork.

**Pull request closed without merging**